.github/workflows/main-pr-ci-cd-frontend.yml

```
@@ -0,0 +1,67 @@
+name: Deploy a new site for each pull request
+
+on:
+  pull_request:
+    branches: [ main ]
+env:
+  PRNUMNER: 'academy${{github.event.pull_request.number}}'
```

```suggestion
  PRNUMNER: 'hyacademypr${{github.event.pull_request.number}}'
```
Cái này phải globally unique nên để ngắn quá sợ sẽ bị trùng